xwiki-platform-core/xwiki-platform-refactoring/xwiki-platform-refactoring-default/src/test/java/org/xwiki/refactoring/internal/DefaultModelBridgeTest.java

```diff
@@ -181,7 +181,7 @@ public void create(MockitoComponentManager mocker) throws Exception
     }
 
     @Test
-    public void createWithException(MockitoComponentManager mocker) throws Exception
+    void createWithException() throws Exception
```

Comment:
these methods really throw exceptions?

Reply:
The call to `getDocument` is throwing an exception

Reply:
ok I'm asking because almost every day I remove "throw Exception" from test methods that don't throw exceptions ;)